helpers/eventwebhook/eventwebhook_test.go

```diff
@@ -0,0 +1,101 @@
+package eventwebhook
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	timestamp      = "1588788367"
+	testPublicKey  = "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEEDr2LjtURuePQzplybdC+u4CwrqDqBaWjcMMsTbhdbcwHBcepxo7yAQGhHPTnlvFYPAZFceEu/1FwCM/QmGUhA=="
+	testPrivateKey = "MHcCAQEEIEQmZgBEh9DcU9zMl34czK3xov4AYvm9P3r7rNB2dNXtoAoGCCqGSM49AwEHoUQDQgAEEDr2LjtURuePQzplybdC+u4CwrqDqBaWjcMMsTbhdbcwHBcepxo7yAQGhHPTnlvFYPAZFceEu/1FwCM/QmGUhA=="
+	signature      = "MEUCIQCtIHJeH93Y+qpYeWrySphQgpNGNr/U+UyUlBkU6n7RAwIgJTz2C+8a8xonZGi6BpSzoQsbVRamr2nlxFDWYNH2j/0="
+)
+
+func generateTestPayload() []byte {
+	payload, _ := json.Marshal(map[string]interface{}{
+		"event":      "test_event",
+		"category":   "example_payload",
+		"message_id": "message_id",
+	})
+	return payload
+}
+
+func TestEventWebhookNewSettings(t *testing.T) {
+	assert.NotNil(t, NewSettings(), "NewSettings() shouldn't return nil")
+}
+
+func TestSetEnableSignedWebhook(t *testing.T) {
+	s := NewSettings()
+	assert.NotNil(t, NewSettings(), "NewSettings() shouldn't return nil")
+
+	s.SetEnableSignedWebhook(true)
+	assert.Equal(t, true, *s.EnableSignedWebhook, fmt.Sprintf("EnableSignedWebhook should be 'true', got %v", *s.EnableSignedWebhook))
+
+	s.SetEnableSignedWebhook(false)
+	assert.Equal(t, false, *s.EnableSignedWebhook, fmt.Sprintf("EnableSignedWebhook should be 'false', got %v", *s.EnableSignedWebhook))
```

Comment:
could use assert.False instead of assert.Equal

Reply:
yes, but this is fine I guess